### PR TITLE
Fix code style

### PR DESCRIPTION
**Pull request by Biont**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
code style

**What is the current behavior?** (You can also link to an open issue here)
Everything is working

**What is the new behavior (if this is a feature change)?**
Everything is hopefully still working since there should have been no functional changes

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

**Other information**:

### CODE REVIEW

File: src/Admin/CacheFlusher.php [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+    public const PURGE_ACTION = 'purge_cache';
 
     public function item(): MenuItem
     {
-        $referer = '&_wp_http_referer=' . urlencode(wp_unslash($_SERVER['REQUEST_URI']));
+        if (isset($_SERVER, $_SERVER['REQUEST_URI'])) {
+            //phpcs:disable WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+            $referer = wp_unslash($_SERVER['REQUEST_URI']);
+        }
+        $referer = '&_wp_http_referer=' . urlencode($referer);
```
Comment (dnaber-de) [inline src/Admin/CacheFlusher.php]: You're refering to `$referer` here that might not be initialized yet depending on the state of `$_SERVER`

File: src/Admin/CacheFlusher.php [hunk 2]
```diff
@@ -24,16 +28,24 @@ public function item(): MenuItem
...
+    /**
+     * phpcs:disable PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+     * @return void
+     */
     public function flush_cache()
     {
-        if (! isset($_GET['_wpnonce']) || ! wp_verify_nonce($_GET['_wpnonce'], self::PURGE_ACTION)) {
+        $wpNonce = filter_input(INPUT_GET, '_wpnonce', FILTER_SANITIZE_STRING);
+        if (
+            !isset($wpNonce)
```
Comment (dnaber-de) [inline src/Admin/CacheFlusher.php]: I think `isset` is the wrong function here, even though it does what you want. However it reads like «if the variable is set ...» but it is always set (initialized) at this point. I would just use `! $wpNonce` of `empty($wpNonce)`

**Review by dnaber-de — APPROVED** [review]
Just found two minor issues where one is just opinionated

File: src/Admin/CacheFlusher.php (hunk 2 above)
Comment (Biont) [reply src/Admin/CacheFlusher.php]: Yes, this looks like an obvious slip. Thanks for spotting this

File: src/Admin/CacheFlusher.php (hunk 1 above)
Comment (Biont) [reply src/Admin/CacheFlusher.php]: Fixed. Thanks. I vaguely recall losing my mind over this, but I forgot why

**Pull request merged**